### PR TITLE
[Python SDK] Adds Global SQL to Python SDK

### PR DESCRIPTION
**Pull request by tonilopezmr**
### :tophat: What is the goal?
* Add Global SQL module in Python SDK with the create database method
* Fix Linter

### :memo: How is it being implemented?
* Add new `GlobalSQL` class calling v1 API Endpoints of SQL services

### Usage
```python
from seaplane import sea

sea.config.set_api_key("_API_KEY_")

database = sea.global_sql.create_database()

database.name
database.username
database.password
```

### :boom: How can it be tested?
* You can run the linter and tests using `npm run lint` command
* Or run only tests by using `npm run tests`

### CODE REVIEW

**Comment by kbknapp** [comment]
Looks like there are a bunch of Compute changes in this PR too, did you mean to commit those? Or do you want to split those into a separate PR?

**Comment by tiago-peres** [comment]
Does that mean I can start using it once merged @tonilopezmr? Thank you for including the Usage 💚🔥

**Comment by tonilopezmr** [comment]
The problem is those changes are related to the `Linter`, I don't know why It didn't throw an error in the CI pipeline before; right now, the CI is not running in the PR.

If I split the changes then the CI won't pass

**Comment by tonilopezmr** [comment]
> Does that mean I can start using it once merged @tonilopezmr? Thank you for including the Usage 💚🔥

You can start using it before to start implementing your part, I'll release a new version of the SDK for you with the unsafe endpoint.

I'm working on documenting the rest of the SDK to let you know more functionalities that the SDK has.

**Comment by kbknapp** [comment]
> If I split the changes then the CI won't pass

It just means you do the Compute changes first in one PR then the SQL. The reason I'd prefer them split is it makes any changelog entries easier to reference and should we ever need to revert it keeps the changes localized to a single functional area.

**Comment by kbknapp** [comment]
> It just means you do the Compute changes first in one PR then the SQL.

(which is something Graphite is perfect for 😜)

**Comment by kbknapp** [comment]
> If I'm using Graphite, how can I do "the Compute changes first in one PR then the SQL." ?

From the current state of your branch, what I'd do is:

- Rebase all the commits of the compute/lints to be at the start of the branch
- Use `gt branch split --by-commit` and pick the first SQL SDK commit to start a new branch
- Then `gt stack sync` to submit the two PRs

If you'd like to hop in a zoom and we can walk through that together just let me know 😄

**Comment by tonilopezmr** [comment]
Thanks!!

**Review by tiago-peres — APPROVED** [review]
LGTM

**Comment by WesleyDRobinson** [comment]
💥 How can it be tested?

    You can run the linter and tests using nox command
    Or run only tests by using nox -s test

File: seaplane-sdk/python/tests/api/test_sql_api.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        add_token_request(requests_mocker)
+
+        def match_authorization(request: Any) -> Any:
+            return (
+                request.headers["Authorization"] == "Bearer This is a token"  # noqa
+                and request.text == "{}"
+            )
+
+        requests_mocker.post(
+            "https://sql.cplane.cloud/v1/tablespaces/provision_tenant",
```
Comment (WesleyDRobinson) [inline seaplane-sdk/python/tests/api/test_sql_api.py:34]: I believe this has been updated to a `GET /v1/databases`

Comment (tonilopezmr) [reply seaplane-sdk/python/tests/api/test_sql_api.py:34]: changing it

**Pull request merged**